guides/v3.7.0/templates/angle-bracket-syntax.md

```diff
@@ -0,0 +1,72 @@
+The [Angle Bracket Syntax](https://github.com/emberjs/rfcs/blob/master/text/0311-angle-bracket-invocation.md) is an alternative style of invoking components in templates. The difference between the Angle Bracket Syntax and the Classic Invocation Syntax is purely syntactical and does not affect the semantics of invoking a component.
```

Comment:
@chrisrng Thanks for writing this! I would like to maybe put this info in a "conversion guide" type section that is outside of `templates`. Maybe something like `reference/syntax-conversion-guide` that could be multi-purpose to other types of syntax. URLs are forever. Since we are teaching angle brackets as the default, we want to avoid a URL structure that makes them seem "new" if that makes any sense. What do you think about moving it under a different topic? Then in places where we use Angle Bracket syntax, we link to this explanation.

Reply:
Sounds good, makes sense!
I've opened a PR to move them there:
https://github.com/cah-danmonroe/guides-source/pull/2

Reply:
Moved.